### DISCUSSION TITLE
Why todo uses admonition-todo

### DISCUSSION THREAD

**Discussion by hwayne** [pradyunsg/furo#459] [Show and tell]
Saw the bit at https://github.com/pradyunsg/furo/blob/main/docs/reference/admonitions.md#todo and had to laugh because I've run into the exact same issue! It's because the [todo extension](https://github.com/sphinx-doc/sphinx/blob/5.x/sphinx/ext/todo.py#L56) manually sets `admonition-todo` as its node class.

**Comment by pradyunsg** [comment]
Hehehe. 

That's the how it happens. The thing to figure out is why that was done in the code. ;)
